docs/metrics.md

```diff
@@ -0,0 +1,142 @@
+
+#### kubevirt_vmi_network_traffic_packets_total
+
+How much packets are being transmitted and received.
+
+| Label | Description |
+|-----------|-------------------------------------------------------------------------------------------------------|
+| name | VMI's name given on it's specification. |
+| namespace | Namespace which the given VMI is related to. |
+| node | Node where the VMI is running on. |
+| interface | Which network interface that packets are being transmitted/received. |
+| type | Whether the packet are being transmitted or received. `tx` when transmitting and `rx` when receiving. |
+
+#### kubevirt_vmi_storage_iops_total
+
+Counter of read and write operations per disk device.
+
+| Label | Description |
+|-----------|----------------------------------------------|
+| name | VMI's name given on it's specification. |
+| namespace | Namespace which the given VMI is related to. |
+| node | Node where the VMI is running on. |
+| drive | Disk device that is being written/read. |
+| type | Whether it's a read or write operation. |
+
+#### kubevirt_vmi_storage_times_ms_total
+
+Total time spent on read and write operations per disk device.
+
+| Label | Description |
+|-----------|----------------------------------------------|
+| name | VMI's name given on it's specification. |
+| namespace | Namespace which the given VMI is related to. |
+| node | Node where the VMI is running on. |
+| drive | Disk device that is being written/read. |
+| type | Whether it's a read or write operation. |
+
+#### kubevirt_vmi_storage_traffic_bytes_total
+
+Total amount of data read and written per disk device.
+
+| Label | Description |
+|-----------|----------------------------------------------|
+| name | VMI's name given on it's specification. |
+| namespace | Namespace which the given VMI is related to. |
+| node | Node where the VMI is running on. |
+| drive | Disk device that is being written/read. |
+| type | Whether it's a read or write operation. |
+
+#### kubevirt_vmi_vcpu_seconds
+
+Total amount of time spent in each vcpu state
+
+| Label | Description |
+|-----------|------------------------------------------------------------------------------------------------------------|
+| name | VMI's name given on it's specification. |
+| namespace | Namespace which the given VMI is related to. |
+| node | Node where the VMI is running on. |
+| id | Indentifier to a single Virtual CPU. |
+| state | Identify the Virtual CPU state. It can be one of libvirtd vcpu's states: `OFFLINE`, `RUNNING` or `BLOCKED` |
+
+#### kubevirt_vmi_phase_count
+
+Total amount of VMIs per node and phase.
+
+| Label | Description |
+|-------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| phase | Phase of the VMI. It can be one of [kubernetes pod lifecycle phases](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/) |
```

Comment:
I would rather refer to https://github.com/kubevirt/kubevirt/blob/master/staging/src/kubevirt.io/client-go/api/v1/types.go#L415.